docs/content/commands/npm-fund.md

```diff
@@ -48,6 +86,21 @@ Show information in JSON format.
 Whether to represent the tree structure using unicode characters.
 Set it to `false` in order to use all-ansi output.
 
+#### workspace
```

Comment:
This should be the same as the definition in `docs/content/using-npm/config.md`

```markdown
#### `workspace`

* Default:
* Type: String (can be set multiple times)

Enable running a command in the context of the configured workspaces of the
current project while filtering by running only the workspaces defined by
this configuration option.

Valid values for the `workspace` config are either: - Workspace names - Path
to a workspace directory - Path to a parent workspace directory (will result
to selecting all of the nested workspaces)

This value is not exported to the environment for child processes.
```

Reply:
And actually, the formatting there is a little bit too fancy for the extremely naive markdown handler in use here.  This patch should fix it: https://gist.github.com/isaacs/d108e87dd0f847d9d108aac7555a301b